src/toady/fetch_service.py

```
@@ -135,3 +140,210 @@ def fetch_review_threads_from_current_repo(
+            elif len(pull_requests) == 1:
+                # Single PR - auto-select it
+                selected_pr = pull_requests[0]
+                self.pr_selector.display_auto_selected_pr(selected_pr)
+                return PRSelectionResult(pr_number=selected_pr.number, cancelled=False)
+
+            else:
+                # Multiple PRs - show interactive selection
+                selected_pr_number = self.pr_selector.select_pr(pull_requests)
+                if selected_pr_number is None:
+                    return PRSelectionResult(pr_number=None, cancelled=True)
+                return PRSelectionResult(pr_number=selected_pr_number, cancelled=False)
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)